python/sglang/srt/managers/io_struct.py

```diff
@@ -107,6 +107,11 @@ def post_init(self):
                     ## suppot batch operation
```

Comment:
"##" -> "#"

Reply:
```suggestion
                    # suppot batch operation
```
You can directly suggest with github btw.